exasol_script_languages_container_ci/cli/commands/run_release.py

```diff
@@ -34,7 +37,9 @@ def run_release(ctx: click.Context,
                 config_file: str,
                 source_repo_url: str,
                 release_id: int,
+                github_token: str,
                 dry_run: bool):
```

Comment:
I didn't mean to use a cli parameter for the GitHub token, only to extract the environment variable access

Reply:
Done